crates/l2/proposer/errors.rs

```diff
@@ -92,6 +93,10 @@ pub enum CommitterError {
     FailedToReExecuteBlock(#[from] EvmError),
     #[error("Committer failed to send transaction: {0}")]
     FailedToSendCommitment(String),
+    #[error("Commiter failed to decode deposit hash")]
+    FailedToDecodeDepositHash,
+    #[error("Commiter failed to merkelize: {0}")]
```

Comment:
nit: 
```suggestion
    #[error("Committer failed to decode deposit hash")]
    FailedToDecodeDepositHash,
    #[error("Committer failed to merkelize: {0}")]
```

Reply:
fixed in bec14139b55734f5f34037cefff5ff577c151829